lib/simp/rake/beaker.rb

```diff
@@ -4,6 +4,31 @@
 require 'fileutils'
 require 'puppetlabs_spec_helper/rake_tasks'
```

Comment:
This is where the problem module-related tasks come from.  

It looks like that could be fixed by just including the beaker tasks, instead:

```suggestion
require 'puppetlabs_spec_helper/tasks/beaker'
```

Reply:
I'll check this out and see if that works better.

Reply:
@op-ct This still ends up dragging everything in anyway. The current solution is the only way that I could find to trim out extra bits.

Reply:
> @op-ct This still ends up dragging everything in anyway.

@trevor-vaughan That sounds like a problem with your test environment—testing from a clean Rakefile, the suggested change doesn't drag in anything extra:

### Gemfile

```ruby
gem 'rake'
gem 'simp-beaker-helpers'    # <--gem updated w/suggested change
gem 'puppetlabs_spec_helper'
gem 'puppet'
```

### Rakefile

```ruby
require 'simp/rake/beaker'
Simp::Rake::Beaker.new(File.dirname(__FILE__))
```

### Output

```
$ bundle exec rake -T
rake beaker                        # Run beaker acceptance tests
rake beaker:run[nodeset]           # Run a Beaker test against a specific Nodeset
rake beaker:sets                   # List available beaker nodesets
rake beaker:ssh[set,node]          # Try to use vagrant to login to the Beaker node
rake beaker:suites[suite,nodeset]  # Run Beaker test suites
rake clean                         # Remove any temporary products
rake clobber                       # Remove any generated files
```

Reply:
@op-ct You were correct, there was another spot in the code in question that was dragging in too much! Modified to match your review.